data/GRND/data.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
validation script is failing because of an error here. Also please format this correctly

Reply:
We went ahead and got both of these corrected. Thanks!

Reply:
I believe I managed to get the deletion to work this time.